docs/blog/introducing-modern-web/index.md

```diff
@@ -1,62 +1,64 @@
 ---
-title: Introducing modern web - Developer tools for modern web development.
+title: Introducing: Modern Web.
 published: false
-description: Reexperience the joy when working with the standards based web. Starting off with a test runner which uses multiple browsers in parallel.
+description: Developer tools for modern web development
```

Comment:
that should be longer as otherwise, it looks "boring" on the overview page

Reply:
reversed description it for now - if anyone has a better description feel free to suggest